Modules/Core/Common/include/itkMath.h

```diff
@@ -329,11 +329,7 @@ FloatAlmostEqual(T                                        x1,
     return false;
   }
 
-  typename Detail::FloatIEEE<T>::IntType ulps = FloatDifferenceULP(x1, x2);
```

Comment:
Are you sure that this has the same behavior, in particular with `-0` and `0`?

Reply:
Thanks for bringing 0 versus -0 to the attention, Matt. In this particular case, `FloatDifferenceULP` returns an integer type. For integer types 0 and -0 are bit-wise equal, in C++. So behavior change, in this particular case.